### PR TITLE
Update validator.jar for html.validation

### PR DESCRIPTION
**Pull request by matthiasblaesing**
6debf7f90cf42e7dcac649cc6d55abf38ab678cc updated the validator.jar, this
update was missing the removal of the CSS validator, which caused
NoClassDefFoundError at runtime:

java.lang.NoClassDefFoundError: org/w3c/css/util/ApplContext
at nu.validator.datatype.MediaQuery.checkValid(MediaQuery.java:47)
at nu.validator.datatype.AbstractDatatype.checkValid(AbstractDatatype.java:77)
at com.thaiopensource.relaxng.impl.DataDataDerivType.dataDeriv(Unknown Source)
at com.thaiopensource.relaxng.impl.PatternMemo.dataDeriv(Unknown Source)
at com.thaiopensource.relaxng.impl.PatternValidator.startElement(Unknown Source)

This changeset updates the validator patch by removing the code for the
css validator.

In addition the libararies were updated to better match the libraries
html validator expects.

### CODE REVIEW

**Comment by matthiasblaesing** [comment]
@neilcsmith-net @junichi11 this is an update/followup of 6debf7f90cf42e7dcac649cc6d55abf38ab678cc, which has reproducible problems (see the new unittest).

**Comment by junichi11** [comment]
@matthiasblaesing I'll test it ASAP. Thank you!

File: ide/html.validation/external/validator-howto-build.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This way the validator-20200626-patched.jar was build:
```
Comment (junichi11) [inline ide/html.validation/external/validator-howto-build.txt:1]: Missing the license header.

```
BUILD FAILED
/home/travis/build/apache/netbeans/scratch/nbbuild/build.xml:2220: Failed Rat test(s):
Unapproved license in 1 file(s) 
https://github.com/apache/netbeans/tree/master/ide/html.validation/external/validator-howto-build.txt
```

**Comment by matthiasblaesing** [comment]
@junichi11 something had to break :-) Pushed an updated version, that holds the short license statement.

**Pull request merged**